app/scripts/modules/cloudfoundry/src/presentation/pipeline/stages/CloudfoundryLoadBalancersExecutionDetails.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { get } from 'lodash';
+
+import {
+  AccountTag,
+  ExecutionDetailsSection,
+  IExecutionDetailsSectionProps,
+  StageExecutionLogs,
+  StageFailureMessage,
+} from '@spinnaker/core';
+
+export function CloudfoundryLoadBalancersExecutionDetails(props: IExecutionDetailsSectionProps) {
+  const { stage } = props;
+  const { context } = stage;
+  const account = get(context, 'credentials', undefined);
+  const region = get(context, 'region', undefined);
+  const loadBalancerNames = get(context, 'loadBalancerNames', undefined);
+  const serverGroupName = get(context, 'serverGroupName', undefined);
+  return (
+    <ExecutionDetailsSection name={props.name} current={props.current}>
+      <div className="step-section-details">
+        <div className="row">
+          <div className="col-md-12">
+            <dl className="dl-horizontal">
+              <dt>Account</dt>
+              <dd>
+                <AccountTag account={account} />
+              </dd>
+              <dt>Region</dt>
+              <dd>
+                {region}
+                <br />
+              </dd>
+              <dt>Server Group Name</dt>
+              <dd>
+                {serverGroupName}
+                <br />
+              </dd>
+              <dt>Load Balancers</dt>
+              <dd>
+                {loadBalancerNames.join(', ')}
+                <br />
+              </dd>
+            </dl>
+          </div>
+        </div>
+      </div>
+      <StageFailureMessage stage={props.stage} message={props.stage.failureMessage} />
+      <StageExecutionLogs stage={props.stage} />
+    </ExecutionDetailsSection>
+  );
+}
+
+// TODO: refactor this to not use namespace
+// eslint-disable-next-line
+export namespace CloudfoundryLoadBalancersExecutionDetails {
+  export const title = 'cloudfoundryLoadBalancersConfig';
+}
```

Comment:
is this even used? if not we could remove this

Reply:
It appears to be used in core because of https://github.com/spinnaker/deck/blob/1ced961c1e71a8003d4ae93aa04dc44932164b07/app/scripts/modules/core/src/domain/IStageTypeConfig.ts#L6.  Removing this from core is non-trivial and we feel that it is beyond the scope of this story.

Reply:
Agree with @stuart-pollock. I've also looked into whether it's optional on prior stages and concluded it isn't yet.